tools/probes/common/check_ruciomover

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python
+# Copyright European Organization for Nuclear Research (CERN) 2017
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Nicolo Magini, <Nicolo.Magini@cern.ch>, 2017
+
+'''
+Probe to check read/write with all protocols on target RSE with rucio client
+'''
+
+import os
```

Comment:
we sort our imports alphabetically

python native imports first
then python native imports with from clause

blank line

python external imports
python external from imports

blank line

rucio imports
rucio imports with from clause